src/controllers/PostsController.ts

```diff
@@ -95,27 +95,38 @@ export default class PostsController {
     res.json(posts);
   }
 
-  public static async update(req: Request, res: Response, next: NextFunction) {
-    const { content, image, AnimalId, TagId } = req.body;
-    const validatedUpdate = await postSchema.validateAsync({
-      content,
-      image,
-      AnimalId,
-      TagId,
-      UserId: req.user?.id,
-    });
-    const [, [updatePost]] = await Post.update(validatedUpdate, {
-      where: { id: req.params.postId, UserId: req.user?.id },
-      returning: true,
-    });
-    if (!updatePost) {
-      throw new CustomError(400, 'you cannot update this post');
+  public static async update(req: Request, res: Response) {
+    try {
+      const { content, image, Animal, Tag, User } = req.body;
```

Comment:
please revert the backend files to their original state, this is not how we deal with roadblocks in frontend